core/src/main/resources/defaults/blueprints/cdp-data-engineering.bp

```diff
@@ -47,7 +47,7 @@
         "serviceConfigs": [
           {
             "name": "yarn_admin_acl",
-            "value": "yarn,hive"
+            "value": "yarn,hive,hdfs"
```

Comment:
We have quite a few default blueprints, shouldn't it be fixed everywhere?

Reply:
Hmm... This is a good question Attila :)
I looked at the other nine default blueprints. Of these, the following configure the YARN service:

1. **cdp-data-engineering**
2. cdp-data-engineering-ha
3. cdp-data-science
4. cdp-data-mart
5. cdp-de-llap
6. cdh6-shared-services
7. cdh6-data-science

But only the cdp-data-engineering blueprint has the yarn_admin_acl configured. If I add the yarn_admin_acl configuration to the other blueprints, I'm not sure what the impact/side effects may be.

Should I go ahead and add it anyway?

Reply:
@ajyoung we should focus on cdp ones. impact/side effects shoyld be figured out on CM side. i think it depends on what are the defaults. (like if the original default(s) would be different from hive/hdfs/yarn...that can have some bad affect)

Reply:
@oleewere @akanto I would like to modify the other four blueprints on a separate commit. That way, if there are unforeseen problems, the other commit can be reverted without reverting this change which is needed to resolve a Blocker Bug (CB-3306). Accordingly, I've opened a separate pull request here: https://github.com/hortonworks/cloudbreak/pull/6160

Reply:
we already have the new one, so ok then